src/aliceVision/mesh/Texturing.cpp

```diff
@@ -451,7 +451,7 @@ void Texturing::generateTexturesSubSet(const mvsUtils::MultiViewParams& mp,
                 if(contrib + 1 == nbContribLevel)
```

Comment:
Simpler and avoid to access outside of the array.
```
if(contrib + 1 == texParams.multiBandNbContrib[band])
{
    ++band;
}
```
So we could remove the variable `nbContribLevel`.